docs/change-log.md

```diff
@@ -2,6 +2,13 @@
 
 This document provides an overview of the changes we are making over time. Feel free to reach out to us in case of any specific questions.
 
+## Product Update October 31st, 2022
```

Comment:
We always publish product updates on Fridays... I know we're a little late this time but I just always set the date to the previous Friday to keep it consistent.